diagnostic.md

```diff
@@ -32,6 +48,26 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
+These are getters:
+$(".btn-two").click(function(){
+     var str = $("p:first").text();
+     alert(str);
+
+     $(".btn-two").click(function(){
+             var str = $("#container").html();
+             alert(str);
+
+
+These are setters:
+$("button").click(function(){
+        $("body").html("<p>Hello World!</p>");
+
+$(document).ready(function(){
+
+  $("button").click(function(){
+      $('input[type="checkbox"]').attr("checked", "checked");
+            });
+        });
```

Comment:
We just want examples of getter and setter methods.  The methods that you invoked are getters (`.text()` and `.html()`) and setters (`.html()` and `.attr()`), but none of those examples will actually run since there are syntax errors (missing braces and parentheses).